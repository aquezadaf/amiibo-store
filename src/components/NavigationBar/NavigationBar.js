import React from 'react';
import { Link } from 'react-router-dom';
import logo from './img/logo.svg';
import styles from './NavigationBar.module.css';
import ShoppingCart from '../../containers/ShoppingCart'

export default function NavigationBar() {
  return (
    <nav className={styles.container}>
      <div>
        <Link to="/">
          <img src={logo} className={styles.image} alt="logo" />
        </Link>
      </div>
      <div>
        <ShoppingCart />
      </div>
    </nav>
  );
}
