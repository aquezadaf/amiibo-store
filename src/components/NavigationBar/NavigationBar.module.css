.container {
    padding: 20px 0 40px 0;
    background: #f8f8f8d0;
    font-size: 16px;
    display: flex;
    justify-content: space-between;
    align-items: center;
    padding: 20px;
}

.image {
    height: 100%;
    object-fit: contain;
    width: 130px;
}
