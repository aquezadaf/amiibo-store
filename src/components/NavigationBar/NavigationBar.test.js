import React from "react";
import { shallow } from "enzyme";
import NavigationBar from "./NavigationBar";

jest.mock("react-router-dom");
jest.mock("../../containers/ShoppingCart", () => () => null);

it("renders navigation bar", () => {
  const wrapper = shallow(<NavigationBar />);
  expect(wrapper.html()).toMatchSnapshot();
});
