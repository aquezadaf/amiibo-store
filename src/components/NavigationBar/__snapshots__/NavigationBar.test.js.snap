// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders navigation bar 1`] = `"<nav class=\\"container\\"><div></div><div></div></nav>"`;
